spec/prog/vm/nexus_spec.rb

```diff
@@ -440,32 +446,32 @@ def new_host(**args)
     end
 
     it "does not match if there is not enough ram capacity" do
-      new_host(total_mem_gib: 1).save_changes
+      new_host(total_mem_gib: 1)
```

Comment:
Nice, I like getting rid of saves

Reply:
That said, if you look at this patch, *most* of it is a opportunistic change to the tests, not having anything to do with the change to nexus.rb, so I'd split it into another patch (but not another PR necessarily, in this I am less particular since I check old PRs much less often than git log with -S or --grep)